jina/clients/request.py

```diff
@@ -94,6 +94,14 @@ def index(*args, **kwargs):
     """Generate a indexing request"""
     yield from _generate(*args, **kwargs)
 
+def update(*args, **kwargs):
+    """Delegates the generator"""
```

Comment:
Maybe something like `"""Generate a update request"""`, to be consistent with the other docstrings?

Reply:
hmm true it should be consistent. Should I update the others?
What we are doing here is to use the `generator delegation` syntax -`yield from generator`. That's why I wrote it there. However, isn't interested in implementation detailes.
I changed it. Thanks